ding/model/template/bc.py

```diff
@@ -173,14 +178,27 @@ def __init__(
                 )
             )
 
-    def forward(self, inputs: Union[torch.Tensor, Dict[str, torch.Tensor]]) -> Dict[str, torch.Tensor]:
+    def forward(self, inputs: Union[torch.Tensor, Dict[str, torch.Tensor]]) -> Dict:
```

Comment:
why delete `Dict[str, torch.Tensor]`

Reply:
That's because I find that when the action space is "reparameterization", the returned type should be ``Dict[str, List]``, which is not compatible to the original doc.